multiregion/docs/multiregion.rst

```diff
@@ -613,6 +633,9 @@ In this section, you will simulate a single broker failure in the ``west`` regio
       Topic: multi-region-default	PartitionCount: 1	ReplicationFactor: 4	Configs: min.insync.replicas=1,confluent.placement.constraints={"version":1,"replicas":[{"count":2,"constraints":{"rack":"west"}}],"observers":[{"count":2,"constraints":{"rack":"east"}}]}
       	Topic: multi-region-default	Partition: 0	Leader: 2	Replicas: 1,2,3,4	Isr: 2	Offline: 1	Observers: 3,4
 
+#. Verify similar replica placement in C3. Note that it may take up to 5 minutes for C3 to properly report the new topic stats.
```

Comment:
```suggestion
#. Verify similar replica placement in |c3|. Note that it may take up to 5 minutes it to properly report the new topic stats.
```

Reply:
@awalther28 note that `C3` is an internal term, but we should use `Control Center` or `Confluent Control Center` in documentation.  

```
.. |c3-short| replace:: Control Center
.. |c3| replace:: Confluent Control Center
```